deephaven/web-client-ui
# feat: Add copy/rename/delete options to notebook overflow menu

* Adds options to make a copy of, rename, or delete the file currently being viewed in the notebook panel
* Resolves #1359

#### Testing Instructions:
* Open any file create a new file
* Navigate to the overflow menu in the top right of the notebook panel
* The "copy file" option should create a copy of the currently open file with all of its contents copied over
    * The created copy should not appear in the file explorer until a save operation is done on it
* The "rename file" option should cause the file to appear in the file explorer with the same contents and specified name
    * If the file was saved under a different name before, the old file should be deleted
    * If the file was never saved before, this operation will be identical to a save
* The "delete file" option should close the file that the notebook panel is currently displaying.
    * If the file has been saved before, the saved file should be deleted and no longer in the file explorer
    * If the file was never saved before, this operation will be identical to closing the file and discarding any changes
